pkg/transparentproxy/kubernetes/kubernetes.go

```diff
@@ -0,0 +1,119 @@
+package kubernetes
+
+import (
+	"fmt"
+	"strconv"
+
+	kube_core "k8s.io/api/core/v1"
+
+	"github.com/kumahq/kuma/pkg/plugins/runtime/k8s/metadata"
```

Comment:
not a fan of having a dependency on `plugins/runtime/k8s/metadata` in this package, but I don't have a good idea how to fix it. Maybe later it will be more clear how to not depend on plugin in `pkg/transparentproxy`

Reply:
Good point, but agree - not clear how to make it better.